charts/harness/Chart.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v2
```

Comment:
Is this chart needed?

Can we simply use the inference-perf container as part of the StepAction, and configure it so that the step actually directly sends the Inference request?

This approach eliminates the chart, along with the need for a separate harness pod (the Tekton step will run as part of some pod anyway).